dashboard/test/controllers/student_snapshots_controller_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'test_helper'
```

Comment:
Optional: It would be awesome to write new tests in [specs syntax](https://github.com/code-dot-org/code-dot-org/blob/staging-next/STYLEGUIDE.md#writing-unit-tests).
It may be a hassle, so I will leave it up to you.